packages/settings/store/middleware.js

```diff
@@ -9,6 +9,7 @@ const settings = store => next => (action) => {
   switch (action.type) {
     case actionTypes.networkConfigSet:
       // store.dispatch(pricesRetrieved());
+      if(!action.data) break;
```

Comment:
Although using `if` condition in a `switch` statement is valid, but it kind of means the cases could be designed better. In your case, `networkConfigSet` is always expected to have data. I'd recommend finding the root cause and fixing it instead of pushing bugs under the rug.